vault/docs/relay.md

```diff
@@ -0,0 +1,9 @@
+# Relay Processes
+
+The Vault must validate the state of the BTC-Relay and ensure it's view is consistent with that of the configured Bitcoin chain.
+
+<p align="center">
+  <img src="./img/relay.png" alt="relay" width="500"/>
```

Comment:
Do we check for initialized status in the loop? That seems weird. Also from the diagram it looks like the result from `Compute start height` is overwritten before it ever gets used?